src/CMakeLists.txt

```diff
@@ -6,20 +6,21 @@ target_sources(timg PRIVATE
 
   buffered-write-sequencer.h buffered-write-sequencer.cc
   display-options.h
-  framebuffer.h     framebuffer.cc
-  image-display.h   image-display.cc
-  image-source.h    image-source.cc
-  iterm2-canvas.h   iterm2-canvas.cc
-  jpeg-source.h     jpeg-source.cc
-  kitty-canvas.h    kitty-canvas.cc
-  renderer.h        renderer.cc
-  terminal-canvas.h terminal-canvas.cc
-  termutils.h       termutils.cc
+  framebuffer.h              framebuffer.cc
+  image-display.h            image-display.cc
+  image-source.h             image-source.cc
+  iterm2-canvas.h            iterm2-canvas.cc
+  jpeg-source.h              jpeg-source.cc
+  slide-image-source.cc      slide-image-source.h
```

Comment:
Can you sort it alphabetically ? So this would be after renderer.h.

Also, put the header first, then the *.cc file, so that it is like the other lines.

Reply:
that was really OCD triggering. (Solved)